package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "Biz-Luminate",
+  "version": "1.0.0",
+  "description": "",
+  "author": "Natalie Montoya, Katie Scruggs, Maria St. Louis-Sanchez, Jesse Packwood, Megan Arellano, Matt Andrews, Thomas Laird",
+  "main": "knexfile.js",
+  "scripts": {
+    "test": "mocha --exit",
+    "server": "nodemon server.js",
+    "client": "cd client && yarn start",
```

Comment:
I was getting an error for yarn start. I changed it locally to npm start to get npm run dev to work.